### DISCUSSION TITLE
How to render pages without the data-server-rendered HTML attribute

### DISCUSSION THREAD

**Discussion by omurad** [nuxt/nuxt#10221] [Questions (Nuxt 2)]
I am using NuxtJS strictly for server-side rendering, without any client-side JS. So the `data-server-rendered` attribute does not serve a purpose in my app.

Is there a built in method to prevent that HTML attribute from being included?

**Comment by danielroe** [comment]
You can use a Nuxt hook to remove it. For example:
```js
nuxt.hook('render:route', (url, result, context) => {
  result.html = result.html.replace(' data-server-rendered="true"', '')
})
```
